Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -270,6 +270,101 @@ Alternatively to validation functions type conversion by using `Integer.parseInt
 * Drozer
 
 
+### Testing Custom URL Schemes
+
+#### Overview
+
+Both Android and iOS allow inter-app communication through the use of custom URL schemes. These custom URLs allow other applications to perform specific actions within the application hosting the custom URL scheme. Much like a standard web URL that might start with `https://`, custom URIs can begin with any scheme prefix and usually define an action to take within the application and parameters for that action.
+
+As a contrived example, consider: `sms://compose/to=your.boss@company.com&message=I%20QUIT!&sendImmediately=true`. Using something like this embedded as a link on a web page, when clicked by a victim on their mobile device, calling the custom URI with maliciously crafted parameters might trigger an SMS to be sent by the vulnerable SMS application with attacker defined content.
+
+For any application, each of these custom URL schemes needs to be enumerated, and the actions they perform need to be tested.
```

Comment:
Maybe should mention that multiple apps can register for any scheme, including their competitors.